src/app/App.js

```diff
@@ -15,12 +13,33 @@ const router = createBrowserRouter([
     children: [
       {
         path: "/",
-        element: <SearchFormWrapper/>
+        element: <SearchFormWrapper />,
+        children: [
+          {
+            path: "/new",
```

Comment:
we can move all `path` into a separate file

Reply:
i hope i have done it right

Reply:
not really, I've left another comment